tests/testthat/test-check_tbl_value_col_ascending.R

```diff
@@ -163,3 +163,22 @@ test_that("(#78) check_tbl_value_col_ascending will sort even if the data doesn'
   expect_equal(nrow(actual), 1)
   expect_equal(actual, expected, ignore_attr = TRUE)
 })
+
+
+test_that("check_tbl_value_col_ascending works when output type IDs differ by target", {
+  hub_path <- test_path("testdata/hub-diff-otid-per-task/")
+  tbl <- arrow::read_csv_arrow(
+    fs::path(hub_path, "model-output/2024-01-10-ILI-model.csv")
+  ) %>%
+    hubData::coerce_to_character()
```

Comment:
shouldn't this be `hubData::connect_model_output()` followed by `hubData::collect_hub()`?

Reply:
I'm honestly not sure.  I have the sense that maybe not, because the connect/collect workflow is only guaranteed to work after the model output file has passed validations, and at this point we're still in the midst of those validations.  But on the other hand, using connect/collect might work just fine after some basic/preliminary checks have passed?

Reply:
Maybe the best thing to do would be to replicate the way it's read in during running validations, which is to use [`read_model_out_file`](https://github.com/hubverse-org/hubValidations/blob/353e8db432ecf03f093d595e116d377400d1235b/R/validate_model_data.R#L67)?

Reply:
Ah... it's actually `hubValidations::read_model_out_file()`

Reply:
once again, I need to refresh before commenting 🙈